docs/reference/migration/migrate_6_0/mappings.asciidoc

```diff
@@ -1,6 +1,17 @@
 [[breaking_60_mappings_changes]]
 === Mapping changes
 
+==== The `_all` meta field is now disabled by default
+
+On new mappings, the `_all` meta field that contains a copy of the text from
+each field is now disabled by default. The `query_string` and
+`simple_query_string` queries that previously used `_all` to search will now
+check if `_all` is enabled/disabled and switch to executing the query across all
+fields if `_all` is disabled. `_all` may be re-enabled by setting `enabled`:
```

Comment:
If you don't specify anything in the `fields` parameter, then `simple_query_string` and `query_string` will automatically try to execute the search across all searchable fields (see: https://github.com/elastic/elasticsearch/pull/20925). If you want to limit it, then you can specify the `"fields"` parameter in the query.

Additionally, you could use `copy_to` and create your own all-type field as a composite of whatever fields you want, and then set that as the `default_field` for the index.